packages/teleterm/src/ui/TopBar/Identity/EmptyIdentityList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ButtonPrimary, Flex, Text } from 'design';
+import { useAppContext } from 'teleterm/ui/appContextProvider';
+
+export function EmptyIdentityList() {
+  const ctx = useAppContext();
+
+  function handleConnect() {
+    ctx.commandLauncher.executeCommand('cluster-connect', {});
+  }
+
+  return (
+    <Flex m="auto" flexDirection="column" alignItems="center">
+      <Text typography="h6" bold mb={2}>
+        Connect with a Cluster
```

Comment:
```suggestion
        Connect to a Cluster
```

I grepped the repo and it seems we use "connect to" rather than "connect with". `packages/teleterm/src/ui/TabHost/ClusterConnectPanel/ClusterConnectPanel.tsx` also says "connect with".

"Connect with X" is fine IMHO when X is another program/tool, like "Connect with Psql" in `packages/teleterm/src/ui/DocumentGateway/Postgres/Postgres.tsx`.

Reply:
Done, thanks for catching this